test/models/FileManagementTableSpec.scala

```diff
@@ -116,7 +116,7 @@ class FileManagementTableSpec extends SpecBase with Generators {
 
       "must return correct body" in {
         table.body mustBe Some(
-          "These are requests you've made but the file is not ready yet. We'll email you when the file is ready."
+          "These are requests you’ve made but the file is not ready yet. We’ll email you when the file is ready."
```

Comment:
I'm not sure about the Welsh translations, but for English translations, we're using two single quotes `''` as an escape character.

Reply:
@yohan-oc Before I raised ticket I spoke to @Joe-smart2 . See comments on ticket. 2 single quotes when displayed on the page show a straight line as opposed to  curly apostrophe. See screenshot on ticket

Reply:
![Screenshot from 2024-11-01 10-06-47](https://github.com/user-attachments/assets/ca964a9c-66b8-4eab-a512-cb2e5b2deaae)
![Screenshot from 2024-11-01 10-05-58](https://github.com/user-attachments/assets/cd998cc6-75eb-4feb-a90b-633745b0f13d)